itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcrossInstances.java

```diff
@@ -692,8 +693,9 @@ public void testBootStrapDumpOfWarehouse() throws Throwable {
         .run("alter database default set dbproperties ('hive.repl.ckpt.key'='', 'repl.last.id'='')");
     try {
       replica.load("", "`*`");
-    } catch (SemanticException e) {
-      assertEquals("REPL LOAD * is not supported", e.getMessage());
+      assertTrue(false);
```

Comment:
This will always fail. Why do you need to have such assertion?

Reply:
assertEquals("REPL LOAD * is not supported", e.getMessage()); - 
Do we not throw this error anymore

Reply:
This particular error is outdated. Earlier, repl load did not raise any exception and hence catch clause was of no use. In this change, Repl load will throw an exception stating that dbname can't be null.